tests/t_json

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```

Comment:
You can use `tests/t_synonyms` as example,
but in your case I suppose function should accept two arguments:
- `sdcv` argument (may be empty)
- expected json output